src/main/java/io/cryostat/net/web/http/api/beta/HttpApiBetaModule.java

```diff
@@ -39,6 +39,12 @@
 
 import io.cryostat.net.web.http.RequestHandler;
 import io.cryostat.net.web.http.api.beta.graph.GraphModule;
+import io.cryostat.net.web.http.api.v2.ProbeTemplateDeleteHandler;
+import io.cryostat.net.web.http.api.v2.TargetProbePostHandler;
+import io.cryostat.net.web.http.api.v2.TargetProbeDeleteHandler;
+import io.cryostat.net.web.http.api.v2.TargetProbesGetHandler;
+import io.cryostat.net.web.http.api.v2.ProbeTemplateUploadBodyHandler;
+import io.cryostat.net.web.http.api.v2.ProbeTemplateUploadHandler;
```

Comment:
Since these handlers have been moved out from the `/api/beta` version space and into `/api/v2`, I think it would also make sense for their `@Provides`/`@Binds` methods to be moved out of the `HttpApiBetaModule` and into `HttpApiV2Module`. What do you think? This way the handlers do not need to be `public` and do not need to be imported by any other class outside of their API version package.

Reply:
Yeh I think this will make more sense to that! I will move them now.

Reply:
If the `public` isn't needed then I would remove it. Most of the time there is no need for a handler's implementation to be accessed directly by any other class, it only needs to be available to the relevant API module so that it can be `@Binds`/`@Provides`'d into the DI graph and registered with the webserver router generically. So the no-keyword, package-private visibility works well.